Locale/ptBR/config.lua

```diff
@@ -1,142 +1,292 @@
+	['TRACKER_SHOW_COMPLETE'] = "Ver missões completas",
+	['TRACKER_SHOW_COMPLETE_DESC'] = "Quando esta caixa está marcada, as missões concluídas são exibidas na ferramenta de rastreamento de missões",
+	['TRACKER_COLOR_OBJECTIVES'] = "Cor do objectivo",
+	['TRACKER_COLOR_OBJECTIVES_DESC'] = "Altere a cor dos objetivos no Rastreador de acordo com a integridade",
+	['TRACKER_COLOR_WHITE'] = "Branco",
+	['TRACKER_COLOR_WHITE_TO_GREEN'] = "Branco ao verde",
+	['TRACKER_COLOR_RED_TO_GREEN'] = "Vermelho ao verde",
+	['TRACKER_FONT_HEADER'] = "Tamanho da fonte para nomes de missões",
+	['TRACKER_FONT_HEADER_DESC'] = "O tamanho da fonte usada pelo rastreamento para nomes de missões",
+	['TRACKER_FONT_LINE'] = "Tamanho da fonte para objectivos",
+	['TRACKER_FONT_LINE_DESC'] = "O tamanho da fonte usada pelo Rastreador para objectivos",
+	['TRACKER_QUEST_PADDING'] = "Preenchimento entre missões",
+	['TRACKER_QUEST_PADDING_DESC'] = "A quantidade de preenchimento entre as missões no Rastreador",
+	['TRACKER_INVALID_LOCATION'] = "Erro: Questie Tracker em um local inválido, redefinir ...",
+	['TRACKER_SORT_OBJECTIVES'] = "Classificando objetivos",
+	['TRACKER_SORT_OBJECTIVES_DESC'] = "Como os objectivos são classificadas no Rastreador",
+	['TRACKER_SORT_BY_COMPLETE'] = "Por %% concluído",
+	['TRACKER_SORT_BY_LEVEL'] = "Por nivel",
+	['TRACKER_SORT_BY_LEVEL_REVERSED'] = "Por nível (invertido)",
+	['TRACKER_DONT_SORT'] = "Não classifique",
+	['TRACKER_LEFT_CLICK'] = "Clique esquerda",
+	['TRACKER_RIGHT_CLICK'] = "Clique direita",
+	['TRACKER_SHIFT'] = "Shift + ",
+	['TRACKER_CTRL'] = "Ctrl + ",
+	['TRACKER_ALT'] = "Alt + ",
+	['TRACKER_DISABLED'] = "Desactivado",
+	['TRACKER_SHORTCUT'] = " Atalho",
+	['TRACKER_SET_TOMTOM_DESC'] = "Atalho para abrir TomTom",
+	['TRACKER_SHOW_QUESTLOG_DESC'] = "O atalho para exibir a missão de missões",
+	
+	--Menu contextual do rastreador
+	['TRACKER_UNFOCUS'] = "Não concentre mais essa missão",
+	['TRACKER_FOCUS_OBJECTIVE'] = "Objetivo do foco ", --o foco permite exibir apenas esta missão /objetivo no mapa
+	['TRACKER_FOCUS_QUEST'] = "Concentre esta missão",
+	['TRACKER_CANCEL'] = "Anular",
+	['TRACKER_UNTRACK'] = "Anular a missâo",
+	['TRACKER_SHOW_QUESTLOG'] = "Visualizar as missões",
+	['TRACKER_SET_TOMTOM'] = "Defina o alvo TomTom",
+	['TRACKER_SHOW_ICONS'] = "Mostrar ícones",
+	['TRACKER_HIDE_ICONS'] = "Ocultar ícones",
+	['TRACKER_OBJECTIVES'] = "Objectivos",
+	['TRACKER_SHOW_ON_MAP'] = "Mostrar no mapa",
+	['TRACKER_UNLOCK'] = "Desbloqueado o rastreador",
+	['TRACKER_LOCK'] =  "Bloqueado o rastreador",
+	
+	--Ordens de Viagem
+	['JOURNEY_TITLE'] = "A Viagem de %s",
+	['JOUNREY_TAB'] = "Minha Viagem",
+	['JOURNEY_ZONE_TAB'] = "Missôes por zonas",
+	['JOURNEY_SEARCH_TAB'] = "Pesquisa avançada",
+	['JOURNEY_AVAILABLE_TITLE'] = "Missôes disponiveis",
+	['JOURNEY_COMPLETE_TITLE'] = "Miss^0es terminadas",
```

Comment:
This looks wrong, should most like be:

```suggestion
	['JOURNEY_COMPLETE_TITLE'] = "Missôes terminadas",
```